artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/ActiveMQDestination.java

```diff
@@ -219,7 +221,12 @@ public static SimpleString createTopicAddressFromName(final String name) {
    }
 
    public static ActiveMQQueue createQueue(final String address) {
-      return new ActiveMQQueue(address);
+
```

Comment:
Hot path here (esp for spring users)

Reply:
I did not mean to change this class. accident! 
fixing it!

Reply:
That one is fixed